documentation/projects/proposals/ingestion_server_removal/20240328-implementation_plan_ingestion_server_removal.md

```diff
@@ -410,52 +429,44 @@ code can all be refactored from
     `end_index` it should handle
   - Use a Sensor to ping the worker's `task/{task_id}` endpoint until the task
     is complete, logging the progress as it goes
-  - Use the
-    [EC2Hook#stop_instances](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/_api/airflow/providers/amazon/aws/hooks/ec2/index.html#airflow.providers.amazon.aws.hooks.ec2.EC2Hook.stop_instances)
-    to shut the instance down. This step should skip in a local environment.
+- Use
+  [`terminate_instance_in_auto_scaling_group`](https://boto3.amazonaws.com/v1/documentation/api/1.26.86/reference/services/autoscaling/client/terminate_instance_in_auto_scaling_group.html)
+  to terminate the instance. Make sure to set `ShouldDecrementDesiredCapacity`
+  to `True` to ensure that the ASG does not try to replace the instance. This
+  task should use the
+  [`NONE_FAILED` TriggerRule](https://airflow.apache.org/docs/apache-airflow/1.10.9/concepts.html#trigger-rules)
```

Comment:
Two notes here:
1. I think we want `NONE_SKIPPED` instead? `NONE_FAILED` would mean those tasks _wouldn't_ run on failures, which I think we want for the step.
2. That link appears to be for Airflow v1.10 😅 

```suggestion
  [`NONE_SKIPPED` TriggerRule](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html#trigger-rules)
```